android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/rsp/RspAccount.kt

```diff
@@ -5,16 +5,18 @@
 
 package com.mytiki.sdk.capture.receipt.capacitor.rsp
 
+import com.google.android.gms.common.internal.AccountType
+import com.mytiki.sdk.capture.receipt.capacitor.Account
+import com.mytiki.sdk.capture.receipt.capacitor.AccountTypeEnum
 import org.json.JSONObject
 
 class RspAccount(
-    private val username: String,
-    private val provider: String,
-    private val verified: Boolean
+    private val account: Account
```

Comment:
If we are going to use the Account.toRsp, it does not need to use an Account object in the constructor. Just use the values directly